components/AppBar/src/ColorThemer/MDCAppBarColorThemer.m

```diff
@@ -0,0 +1,26 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCAppBarColorThemer.h"
+
+@implementation MDCAppBarColorThemer
+
++ (void)applyColorScheme:(NSObject<MDCColorScheme> *)colorScheme
+                toAppBar:(MDCAppBar *)appBar {
+  appBar.navigationBar.backgroundColor = colorScheme.primaryColor;
```

Comment:
With app bar, this will need to also affect the flexible header view's color as they are independent layered views